mod-audit/src/main/resources/global/destroyTest.feature

```diff
@@ -41,7 +41,62 @@ Feature: remove all test data
     Given path 'users', userid
     When method DELETE
 
+  @DeleteUserGroup
+  Scenario: Delete user group
+    Given path 'groups', userGroupId
+    When method DELETE
+
   @DeleteServicePoint
   Scenario: Delete user
     Given path 'service-points', servicePointId
+    When method DELETE
```

Comment:
There is execution of destroy-data.feature at tests. So it looks that it is not necessary to remove test data unless there is special case for such cleaning.

Reply:
Remove test data is required due to the invocation of initTest.feature before each scenario.